### PR TITLE
CB-13358 SharedServiceConfigsView filled in every cases

### PR DESCRIPTION
**Pull request by afarsang**
See detailed description in the commit message.

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/DatalakeService.java [hunk 1]
```diff
@@ -136,11 +136,23 @@ private SharedServiceConfigsView createSharedServiceConfigView(Stack stack) {
             sharedServiceConfigsView.setRangerAdminPort(DEFAULT_RANGER_PORT);
             sharedServiceConfigsView.setDatalakeCluster(false);
             sharedServiceConfigsView.setAttachedCluster(true);
+        } else if (blueprintUtils.isSharedServiceReadyBlueprint(blueprint)) {
```
Comment (doktoric) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/DatalakeService.java:139]: qq should we just check the type of the stack ?

**Review by pdarvasi — APPROVED** [review]

File: core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/DatalakeService.java (hunk 1 above)
Comment (pdarvasi) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/DatalakeService.java:139]: +1

Comment (afarsang) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/DatalakeService.java:139]: should. but this is how it was, I can rewrite if you insist

Comment (doktoric) [reply core/src/main/java/com/sequenceiq/cloudbreak/service/sharedservice/DatalakeService.java:139]: we need 3 branch here 
WORKLOAD  -> this is an attached cluster
DATALAKE -> this is a datalake
LEGACY ->  not attached to any datalake or environment

**Review by doktoric — APPROVED** [review]

**Pull request merged**